src/garage/torch/utils.py

```diff
@@ -76,3 +78,21 @@ def update(m, name, param):
                 update(named_modules[module_name], param_name, new_param)
         else:
             update(module, name, new_param)
+
+
+def from_numpy(array):
```

Comment:
is this really clearer than the torch equivalent?

Reply:
In parallel I'm working on a pr for torch utils. It adds the necessary torch utility functions and their proper documentation. The reason that we've added functions like this is because there is no way of defining a global gpu mode in torch, so we have this class, and wrapper functions that automatically move tensors over to gpu if a user has specified `torch_utils.set_gpu(True)`

Reply:
does this need be in this PR, or could it appear in that PR or a separate one?

Reply:
i.e. would removing it from this PR break something, or merely remove a feature?

Reply:
Sorry ignore what I said above, it didn't answer your question. You're right it is not clearer than the pytorch doc. In my upcoming torch utils pr, I've updated docstrings to use their torch equivalents.

Reply:
sorry, i was not asking that. i was asking, "is making this helper function really clearer than just using its one-line implementation everywhere?"

Reply:
I see your point. I think that its correct to not make functions for one liners. Can I make this fix in a subsequent pr?

Reply:
file an issue and assign yourself. put it in the 2020.02 milestone.

Reply:
Wait, this function automatically handles setting the right device. It's definitely helpful for decreasing the number of pain points when moving an algorithm to GPU.

I don't want to see hundreds of "one liners" using `garage.torch.utils._DEVICE` everywhere.